website/archivedVersions.json

```diff
@@ -1,4 +1,8 @@
 {
+  "28.x": "https://jest-archive-august-2023.netlify.app/docs/28.x/getting-started/",
+  "27.x": "https://jest-archive-august-2023.netlify.app/docs/27.x/getting-started/",
+  "26.x": "https://jest-archive-august-2023.netlify.app/docs/26.x/getting-started/",
+  "25.x": "https://jest-archive-august-2023.netlify.app/docs/25.x/getting-started/",
```

Comment:
These urls could be "cleaned" once we know where to store the archives. 
Related to the process here: https://github.com/facebook/react-native-website/issues/3819

Note: if Meta adopts a given archiving process for React-Native, OpenJS remains free to adopt another process if needed.
For now many things are still under Meta's accounts:
- Jest Netlify sites: prod, v1 archive, august-2023 archive
- Jest Crowdin account

Reply:
We should resolve this, thanks for bringing it up!